lib/electr/repl/reader.rb

```diff
@@ -9,6 +9,7 @@ def initialize(readline_lib = Readline)
 
     def run
       line = @readline.readline(@prompt, true)
+      return if line.nil?
```

Comment:
This trick is the only one needed, isn't it? There is no need to modify the code in `repl.rb`. Because readline returns nil on an end of file one can surely do:

    line or exit

and that's it! Actually that cannot be a direct call to `exit` because we have to say «Bye» like for ctrl-c but this is the idea. What do you think?